deploy-manage/cloud-organization/billing/elasticsearch-billing-dimensions.md

```diff
@@ -20,40 +24,34 @@ The number of VCUs you need is determined by:
+* **Machine learning trained model autoscaling**: [Trained model autoscaling](/deploy-manage/autoscaling/trained-model-autoscaling.md) is always enabled and cannot be disabled, ensuring efficient resource usage, reduced costs, and optimal performance without manual configuration.
 
   Trained model deployments automatically scale down to zero allocations after 24 hours without any inference requests. When they scale up again, they remain active for 5 minutes before they can scale down. During these cooldown periods, you will continue to be billed for the active resources.
- 
-* **Indexing Strategies:** Consider your indexing strategies and how they might impact overall VCU usage and costs:
-  
-    * To ensure optimal performance and cost-effectiveness for your project, it’s important to consider how you structure your data.
-        * Consolidate small indices for better efficiency. We recommend avoiding a design where your project contains hundreds of very small indices, specifically those under 1GB each.
-    * Why is this important?
-         * Every index in Elasticsearch has a certain amount of resource overhead. This is because Elasticsearch needs to maintain metadata for each index to keep it running smoothly. When you have a very large number of small indices, the combined               overhead from all of them can consume more CPU resources than if the same data were stored in fewer, larger indices. This can lead to higher resource consumption and hence higher costs and potentially impact the overall performance of your project.
-
-    * Recommended Approach
-        * If your use case naturally generates many small, separate streams of data, we advise implementing a process to consolidate them into fewer, larger indices. This practice leads to more efficient resource utilization. By grouping your data               into larger indices, you can ensure a more performant and cost-efficient experience with Elasticsearch Serverless.
+* **Indexing strategies** Consider your indexing strategies and how they might impact overall VCU usage and costs:
+  * To ensure optimal performance and cost-effectiveness for your project, it's important to consider how you structure your data. Consolidate small indices for better efficiency. In general, avoid a design where your project contains hundreds of very small indices, specifically those under 1GB each.
+  * Why is this important? Every index in {{es}} has a certain amount of resource overhead. This is because {{es}} needs to maintain metadata for each index to keep it running smoothly. When you have a very large number of small indices, the combined overhead from all of them can consume more CPU resources than if the same data were stored in fewer, larger indices. This can lead to higher resource consumption and hence higher costs and potentially impact the overall performance of your project.
```

Comment:
a bullet explaining a previous bullet seems like an anti-pattern. would consider restructuring this further, probably to use paragraphs instead of child bullets